index.css

```diff
@@ -60,10 +68,11 @@ body {
 
 #main {
```

Comment:
- [ ] Good job implementing CSS Flexbox in the `main` content. I suggest changing it to a `section` tag to follow HTML semantic tag requirements. 😄